client/post-editor/editor-drawer/index.jsx

```diff
@@ -12,7 +12,7 @@ import Accordion from 'components/accordion';
 import AccordionSection from 'components/accordion/section';
 import Gridicon from 'components/gridicon';
 import CategoriesTagsAccordion from 'post-editor/editor-categories-tags/accordion';
-import EditorSharingAccordion from 'post-editor/editor-sharing/accordion';
+import EditorSharingAccordion from 'async-component!post-editor/editor-sharing/accordion';
```

Comment:
>I would probably split the `async-component!` loader syntax up into another PR, since IMHO it adds a fair bit of complexity.

What would you suggest be included in the pull request?